src/x86/avx2.rs

```diff
@@ -621,6 +687,7 @@ pub fn _mm256_sll_epi64(a: i64x4, count: i64x2) -> i64x4 {
 /// shifting in zeros, return the results;
 #[inline(always)]
 #[target_feature = "+avx2"]
+#[cfg_attr(test, assert_instr(vpsllw))] // TODO: should this be pslli
```

Comment:
@BurntSushi do you know what's up with this? (and the two below)

Reply:
It looks like Clang's codegen tests think that `pslli` should be emitted, at least in the llvm IR: https://github.com/llvm-mirror/clang/blob/6d55fdd9ee95cdb0e6fd34df57e044faff8fe5f0/test/CodeGen/avx2-builtins.c#L988-L989

Reply:
But it looks like the llvm codegen tests agree with what we're observing, and specifically that pslli in the ir gets lowered to vpsllw: https://github.com/llvm-mirror/llvm/blob/4c92030df73fdfac8b57d86241d26796f7f35a80/test/CodeGen/X86/avx2-intrinsics-fast-isel.ll#L2707-L2754